tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/generalize_zip_codes_test.py

```diff
@@ -126,9 +129,13 @@ def test_generalize_zip_codes_cleaning(self):
                 (1006, 1006, 0, datetime.strptime('2020-08-30',
                                                   '%Y-%m-%d').date(), 1006,
                  'StreetAddress_PIIZIP', 1585250, '235**'),
+                # 1007: `UpdateInvalidZipCodes` replaces this type of value
+                # with `Response removed due to invalid value` before
+                # `GeneralizeZipCodes` runs. So, '23512-4' here does not
+                # appear in the real world.
                 (1007, 1007, 0, datetime.strptime('2020-08-30',
                                                   '%Y-%m-%d').date(), 1007,
-                 'StreetAddress_PIIZIP', 1585250, '235****'),
+                 'StreetAddress_PIIZIP', 1585250, '23512-4'),
```

Comment:
@hiro-mishima thanks for the update, this case should also work fine since we have the 5 digits available in the first part of the zip code value we should be generalizing it to 123** format.